src/nav/style-guide.yml

```diff
@@ -13,6 +13,8 @@ pages:
         path: /docs/style-guide/writing-strategies/organize-doc
       - title: How to write introductions
         path: /docs/style-guide/writing-strategies/how-to-write-intros
+      - title: Choose from our install options
```

Comment:
Not sure I'd place it here. Maybe in the Writing docs section?

Reply:
The reason I put it here is that I think installation is such a crucial part of our docs. But I guess I'm not ultimately not opposed to moving it.

Reply:
Unfortunately, there just isn't a good, relevant category in the Writing docs section. This doc aligns more, at least in spirit, with the other docs in the writing strategies section, I think.